ospfd/ospf_te.c

```diff
@@ -1551,6 +1554,10 @@ ospf_mpls_te_lsa_refresh (struct ospf_lsa *lsa)
   else
     top = ospf_lookup ();
 
+  if (OSPF_DEBUG_TRACE)
```

Comment:
Why is this debug not IS_DEBUG_OSPF (lsa, LSA_GENERATE) and why not just expand the debug immediately below?